bedrock/test.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+var bedrock = require('bedrock');
+require('../lib/main.js');
```

Comment:
Could also simplify to `require('..')` and let it resolve via `main` field in `package.json`.